Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:10-alpine
+WORKDIR /app
+COPY . .
+
+# Install system packages
+RUN apk update --no-cache \
+  && apk add --no-cache \
+    ca-certificates \
+    build-base \
+    python3 \
+    git
+
+# Upgrade pip and install Pipenv
+RUN pip3 install --upgrade pip \
+	&& pip install pipenv
+
+# Install Leon
+RUN npm install
+RUN npm run build
+RUN npm run postinstall
+
```

Comment:
add **RUN cd ./bridges/python/ && pipenv install && cd /app** to install Python dependencies

Reply:
done